apps/ios/Shared/Views/Chat/Group/GroupMemberInfoView.swift

```diff
@@ -368,6 +379,24 @@ struct GroupMemberInfoView: View {
         }
     }
 
+    private func blockMemberButton(_ mem: GroupMember) -> some View {
```

Comment:
state update is not observed until sheet is reopened
can be ok to just close sheet